db/helpers.js

```diff
@@ -24,7 +24,10 @@ const loadAssignments = (dirname, cb, exclude = []) => {
   })
 }
 
+const classMixin = (A, B) => class A extends B {}
+
 module.exports = {
   mapJSFiles,
-  loadAssignments
+  loadAssignments,
+  classMixin
```

Comment:
Why do you define the `classMixin` twice?

Reply:
`db` is a module totally separate from the `api`. So it must depend only in its own helpers. But, we do not need the `classMixin` for the moment.